test/controllers/recirculating_infusion_mash_systems_controller_test.rb

```diff
@@ -1,7 +1,47 @@
 require 'test_helper'
 
 class RecirculatingInfusionMashSystemsControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
+  setup do
+    @user = users(:ohm)
+    log_in_as(@user)
+    @rhizome = rhizomes(:zeus)
+    @rims = recirculating_infusion_mash_systems(:rims_two)
+    @rims.set_rims
```

Comment:
There's no set_rims function on the model - are you thinking of the private method in the controller? Not sure what you're going for here since you've already set ```@rims``` to your ```:rims_two``` fixture.